src/iterative_ensemble_smoother/ies.cpp

```diff
@@ -13,9 +13,15 @@ using Eigen::MatrixXd;
 using Eigen::VectorXd;
 namespace py = pybind11;
 
-enum struct Inversion { exact = 0, subspace_exact_r = 1, subspace_re = 3 };
+enum struct Inversion
```

Comment:
Are you manually re-formatting here or did you run clang-format or something like that?
If you are manually re-formatting, please run clang-format instead. 
You can do this by executing the script clang-format that's in the script folder.

I have added an issue to run clang-format as part of CI #89

Reply:
It was automatically re-formatted. I will take a look and run the script.